### ISSUE TITLE
`--noUnusedLocals` does not work at top-level of commonJS module

### ISSUE THREAD

**Issue by andy-ms** [Microsoft/TypeScript#17221]
**TypeScript Version:**  nightly (2.5.0-dev.20170712)

**Code**

```ts
const a = 1;
const b = 2;
{
	const c = 3;
}

module.exports = { a };
```

**Expected behavior:**

Errors for both `b` and `c`.

**Actual behavior:**

Only error for `c`.

**Comment by somithajoseph** [comment]
Is this the same as previously fixed https://github.com/Microsoft/TypeScript/issues/11324 ? I am still getting the  issue mentioned [here](https://github.com/Microsoft/TypeScript/issues/11324) with typescript 2.6.1 version and latest nightly build 2.7.0-dev.20171102

**Comment by mhegazy** [comment]
> Is this the same as previously fixed #11324 

no. this is a different one.

> I am still getting the issue mentioned at #11324 with typescript 2.6.1 version and latest nightly build 2.7.0-dev.20171102

please file a new issue and give us some context to be able to investigate.

**Comment by somithajoseph** [comment]
Done. https://github.com/Microsoft/TypeScript/issues/19700
